omod/src/test/java/org/openmrs/module/aijar/page/controller/MarkPatientDeadTest.java

```diff
@@ -0,0 +1,126 @@
+package org.openmrs.module.aijar.page.controller;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.Concept;
+import org.openmrs.Patient;
+import org.openmrs.api.context.Context;
+import org.openmrs.web.test.BaseModuleWebContextSensitiveTest;
+
+import java.util.Calendar;
+import java.util.Date;
+
+/**
+ * Created by lubwamasamuel on 19/15/2016.
+ */
+public class MarkPatientDeadTest extends BaseModuleWebContextSensitiveTest {
+
+    Patient patient = new Patient();
+    Concept concept = new Concept();
+    Date date = new Date();
+
+
+    @Before
+    public void setRequirementsForTest() {
+        patient = Context.getPatientService().getPatient(2);
+        concept = Context.getConceptService().getConcept("unknown");
+    }
+
+
+    private Date modifyPatientBirthDate(int daysOff,Date date){
```

Comment:
@slubwama The test data set is fixed and will not change, so this method is not needed. use actual dates, and a comment to show the birth date for the person reading this function later